.github/workflows/nightly-build.yml

```diff
@@ -9,14 +9,34 @@ jobs:
     runs-on: ubuntu-latest
     name: Verify latest commit
     outputs:
-      RUN_BUILD: ${{ steps.verify_commit.outputs.RUN_BUILD }}
+      RUN_BUILD: ${{ steps.set_output.outputs.RUN_BUILD }}
```

Comment:
I am not getting this change. There is no such step `set_output` in this workflow.
Did you test the changed workflow on your fork?
There once was a KT session in our team about testing workflow changes on fork.
You can ask others, at least @jan-vcapgemini will know...

Reply:
Maybe this helps: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs